solutions/src/main/java/ru/sberbank/school/task06/CountMap.java

```diff
@@ -0,0 +1,77 @@
+package ru.sberbank.school.task06;
+
+import java.util.Map;
+
+/**
+ * Интерфейс мапы, которая ведет подсчёт вхождений элемента в контейнер.
+ * <p>
+ * <p>
+ * Пример(без дженериков):
+ * <p>
+ * CountMap map = new ... ();
+ * <p>
+ * map.add("Some word");
+ * <p>
+ * map.add("Some word");
+ * <p>
+ * <p>
```

Comment:
&lt;p&gt; tag should be preceded with an empty line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-181929521>*